src/bsoncxx/builder/stream/value_context.hpp

```diff
@@ -60,8 +60,11 @@ class value_context {
 
     operator single_context();
 
+#if !defined(_MSC_VER)
```

Comment:
@markbenvenuto Do you think it matters here? I'm not sure. Nobody is going to build with any MSVC earlier than 2015 since it just won't work, and we are hoping to remove these once the bugs are fixed in 2015